crates/forge/src/pretty_printing.rs

```diff
@@ -41,7 +41,11 @@ pub(crate) fn print_test_seed(seed: u64) {
     println!("{}: {seed}", style("Fuzzer seed").bold());
 }
 
-pub(crate) fn print_test_result(test_result: &TestCaseSummary, fuzzer_runs: Option<u32>) {
+pub(crate) fn print_test_result(
+    test_result: &TestCaseSummary,
+    fuzzer_runs: Option<u32>,
+    fuzzer_seed: Option<u64>,
```

Comment:
I understand the logic that we only print manually configured seed here, but I think we should always print the seed after a fuzzed test, no matter if configured globally or in code. We can then get rid of printing the seed at the end of the execution (and of this weird `was_fuzzed` variable I introduced 😅)

Therefore we should pass a struct here I think - either we fuzzed or not. There should be no case when one of those args is None and the other is not

Reply:
I think we can just return `seed` from `run_with_fuzzing`, but that's up to u how to solve it